optuna/study/study.py

```diff
@@ -786,13 +789,18 @@ def objective(trial):
                 Parameter values to pass your objective function.
             user_attrs:
                 A dictionary of user-specific attributes other than ``params``.
+            skip_if_exists:
+                When :obj:`True`, prevents duplicate trials from being enqueued again.
```

Comment:
Not a strong opinion, but how about updating a docstring to describe the method may accept duplicated parameters if the same parameters are enqueued at the same time?

Reply:
8a3bbc13da7e54592a1f7e18251df43c88ec2a4c :+1: